internal/cluster/sbd.go

```diff
@@ -68,13 +68,19 @@ func FindMatches(pattern string, text []byte) map[string]interface{} {
 	r := regexp.MustCompile(pattern)
 	values := r.FindAllStringSubmatch(string(text), -1)
 	for _, match := range values {
-		key := strings.Replace(match[1], " ", "_", -1)
+		key := strings.ReplaceAll(match[1], " ", "_")
 		if _, ok := configMap[key]; ok {
 			switch configMap[key].(type) {
 			case string:
-				configMap[key] = []interface{}{configMap[key]}
+				{
```

Comment:
Why this brackets? Is the linter doing it?
It looks uglier hehe

Reply:
Yes :D 😆